bindings/rust/s2n-tls/src/config.rs

```diff
@@ -36,6 +36,14 @@ impl Config {
     /// Returns a Config object with pre-defined defaults.
     ///
     /// Use the [`Builder`] if custom configuration is desired.
+    ///
+    /// # Safety:
```

Comment:
Generally `# Safety` is for `unsafe fn`s. It might be better to do `Note` or `Warning`?

Reply:
I'll go with Warning. Note doesn't seem strong enough.